src/btree/row_modify.c

```diff
@@ -425,22 +422,11 @@ __wt_update_obsolete_check(
      */
     if (count > WT_THOUSAND) {
         WT_STAT_CONN_INCR(session, cache_eviction_force_long_update_list);
-        __wt_page_evict_soon(session, cbt->ref);
+        __wt_page_evict_soon(session, ref);
```

Comment:
Is this a bug fix? It seems to be the only reason the WT_REF is passed in, so I wonder if we could keep the old behavior?

Reply:
No, I reverted to original code.